BoxContentSDK/BoxContentSDK/Helper/BOXRepresentationsHelper.m

```diff
@@ -0,0 +1,99 @@
+//
+//  BOXRepresentationsHelper.m
+//  BoxContentSDK
+//
+//  Created by Prithvi Jutur on 8/24/18.
+//  Copyright © 2018 Box. All rights reserved.
+//
+
+#import "BOXRepresentationsHelper.h"
+#import "BOXContentSDKConstants.h"
+
+@interface BOXRepresentationsHelper()
+
+///**
+// Setting a representation or list option will include availability of file with request representation information
+// */
+//@property (nonatomic, readwrite, strong) NSMutableOrderedSet *representationsRequested;
+
+/**
+ Ordered list of representation types, returns first representation type supported by the specified file type
+ */
```

Comment:
The property is a boolean. but the comment says "ordered lists" which is confusing. We should improve the clarity of this comment.